basics/installation/install-with-docker.md

```diff
@@ -0,0 +1,471 @@
+
+docker compose will help starting and stopping your PrestaShop stack, persist your database with a volume, persist your installation, and will allow you to mount local modules and/or themes in PrestaShop. 
+
+### Create the docker compose.yml manifest
+
+```yaml
+version: '3'
+services:
+  mysql:
+    container_name: some-mysql
+    image: mysql:5.7
+    restart: unless-stopped
+    environment:
+      MYSQL_ROOT_PASSWORD: admin
+      MYSQL_DATABASE: prestashop
+    networks:
+      - prestashop_network
+  prestashop:
+    container_name: prestashop
+    image: prestashop/prestashop:latest
+    restart: unless-stopped
+    depends_on:
+      - mysql
+    ports:
+      - 8080:80
+    environment:
+      DB_SERVER: some-mysql
+      DB_NAME: prestashop
+      DB_USER: root
+      DB_PASSWD: admin
+    networks:
+      - prestashop_network
+networks:
+    prestashop_network:
+```
+
+{{% notice info %}}
+When using a M1-chip Mac, you may add: `platform: linux/x86_64` on each container declaration. 
+{{% /notice %}}
+
+### Automatically install PrestaShop with test data
+
+{{% notice note %}}
+This is the easiest and quickest way to start and test PrestaShop with Docker.
+{{% /notice %}}
+
+You can automatically install PrestaShop, and add test data, without having to use the installation assistant when creating your Docker compose stack. 
+
+To achieve this, you need to set those environment variables in your PrestaShop container declaration in your  `docker-compose.yml` file:
+
+```yaml
+PS_DEMO_MODE: 1
+PS_INSTALL_AUTO: 1
+PS_DOMAIN: localhost:8080
+```
+
+Then run `docker compose up` in your terminal, wait a few minutes, and you will be able to access a PrestaShop instance on `http://localhost:8080` with demo products, and `http://localhost:8080/admin` for the administration side.
+
+Complete docker compose manifest for reference:
+
+```yaml
+version: '3'
+services:
+  mysql:
+    container_name: some-mysql
+    image: mysql:5.7
+    restart: unless-stopped
+    environment:
+      MYSQL_ROOT_PASSWORD: admin
+      MYSQL_DATABASE: prestashop
+    networks:
+      - prestashop_network
+  prestashop:
+    container_name: prestashop
+    image: prestashop/prestashop:latest
+    restart: unless-stopped
+    depends_on:
+      - mysql
+    ports:
+      - 8080:80
+    environment:
+      DB_SERVER: some-mysql
+      DB_NAME: prestashop
+      DB_USER: root
+      DB_PASSWD: admin
+      PS_DEMO_MODE: 1
+      PS_INSTALL_AUTO: 1
+      PS_DOMAIN: localhost:8080
+    networks:
+      - prestashop_network
+networks:
+    prestashop_network:
+```
+
+### Manually install PrestaShop with Installation Assistant, and begin testing
+
+To manually install PrestaShop using the Installation Assistant, don't set the environment variables `PS_DEMO_MODE` and `PS_INSTALL_AUTO`.
+
+In the directory where the docker compose.yml file is located, run: 
+
+- Start the stack: 
+
+```
+docker compose up
+```
+
+- Start the stack, in background: 
```

Comment:
Shall we start the stack or start the stack in the background? What's the difference? Maybe it should be something like:
You can also start your stack in the background, if you do that then something something